Library/Homebrew/cmd/pull.rb

```diff
@@ -1,30 +1,41 @@
 # Gets a patch from a GitHub commit or pull request and applies it to Homebrew.
 # Optionally, installs the formulae changed by the patch.
 #
-# Usage: brew pull [options...] <patch-source>
+# Usage: brew pull [options...] <patch-source> [<patch-source> ...]
 #
-# <patch-source> may be any of:
-#   * The ID number of a pull request in the Homebrew GitHub repo
-#   * The URL of a pull request on GitHub, using either the web page or API URL formats
+# Each <patch-source> may be one of:
+#   * The ID number of a PR (Pull Request) in the homebrew/core or legacy-homebrew
+#       GitHub repo
+#   * The URL of a PR on GitHub, using either the web page or API URL
+#       formats. In this form, the PR may be on homebrew/brew, homebrew/core, or
+#       any tap.
 #   * The URL of a commit on GitHub
 #   * A "brew.sh/job/..." string specifying a testing job ID
 #
 # Options:
-#   --bottle:    Handle bottles, pulling the bottle-update commit and publishing files on Bintray
-#   --bump:      For one-formula PRs, automatically reword commit message to our preferred format
-#   --clean:     Do not rewrite or otherwise modify the commits found in the pulled PR
+#   --bottle:      Handle bottles, pulling the bottle-update commit and publishing files on Bintray
+#   --bump:        For one-formula PRs, automatically reword commit message to our preferred format
+#   --clean:       Do not rewrite or otherwise modify the commits found in the pulled PR
 #   --ignore-whitespace: Silently ignore whitespace discrepancies when applying diffs
-#   --install:   Install changed formulae locally after pulling the patch
-#   --resolve:   When a patch fails to apply, leave in progress and allow user to
-#                 resolve, instead of aborting
+#   --resolve:     When a patch fails to apply, leave in progress and allow user to
+#                  resolve, instead of aborting
 #   --branch-okay: Do not warn if pulling to a branch besides master (useful for testing)
-#   --legacy:    Pull legacy formula PR from Homebrew/homebrew
-#                (TODO remove it when it's not longer necessary)
+#   --legacy:      Pull legacy formula PR from Homebrew/homebrew
+#                  (TODO remove it when it's not longer necessary)
+#   --no-pbcopy:   Do not copy anything to the system clipboard
+#   --no-publish:  Do not publish bottles to bintray
```

Comment:
World's smallest nit: Should possibly be `Bintray` rather than `bintray` given it's a trademark.